src/main/groovy/edu/oregonstate/mist/locations/MergeUtil.groovy

```diff
@@ -162,5 +163,17 @@ class MergeUtil {
             }
         }
 
+        def missingLocations = []
+        campusMapData.each { key, value ->
+            if(!data.find { it.id == value.id }) {
+                missingLocations.add(value.id)
+            }
+        }
+        float ratioMissing = missingLocations.size() / campusMapData.size()
+        if(ratioMissing < MISSING_LOCATIONS_THRESHOLD) {
```

Comment:
Could you add a space between if and ( in your if statements to match the style of the other if statements in this file?